serde_with/src/guide/feature_flags.md

```diff
@@ -6,17 +6,19 @@ Each entry will explain the feature in more detail.
 `serde_with` is fully `no_std` compatible.
 Using it in a `no_std` environment requires using `default-features = false`, since `std` is enabled by default.
 
-1. [`alloc`](#alloc)
-2. [`std` (default)](#std-default)
-3. [`base64`](#base64)
-4. [`chrono_0_4`](#chrono_0_4)
-5. [`guide`](#guide)
-6. [`hex`](#hex)
-7. [`indexmap_1`](#indexmap_1)
-8. [`indexmap_2`](#indexmap_2)
-9. [`json`](#json)
-10. [`macros` (default)](#macros-default)
-11. [`time_0_3`](#time_0_3)
+- [Available Feature Flags](#available-feature-flags)
```

Comment:
Can you please leave the features as a numbered list.

Reply:
Damn, I manually renumbered this, I must have some mark down formatter running on save.